### DISCUSSION TITLE
No SMTP servers were available for XXXX

### DISCUSSION THREAD

**Discussion by elmahdilaarej** [postalserver/postal#1787] [Help with using Postal] [answered]
Hi  i hope you guys are doing well basically i get the No SMTP servers were available for XXXX issue, i did check online and seems that this issue happen because digitalocean block port 25, so i did contact support to ask them to open the port 25, they said that my account is not restricted and all ports are not blocked so i did the following query :
`nmap -p my ip address 25`
and i can see that the port 25 is open so looks like that this is not the issue im facing.
what i did check is to change the connection from ipv6 to ipv4 which i change also.
I did  check the supertool and my domain seems good. I tried multiple options but seems nothing working for my case, could you please help me?
Thanks

**Comment by willpower232** [comment, marked as answer]
This error means that whilst your port 25 is open, something is preventing your Postal from connecting to other email services port 25s.

Do you have an outbound firewall in place somehow?

**Reply by elmahdilaarej** [reply]
yeah i did contact again digitalocean and it seems that they are blocking the port  25 even my account is pretty old, do you guys suggest/know any other company that provide vps with open port 25 ? any suggestion is pretty much appreciate it

**Reply by willpower232** [reply]
I'm not sure, you'll have to ask them how it can be done. Some will say no and some will say you have to ask for permission.

**Reply by elmahdilaarej** [reply]
exactly! it turn out that digitalocean is blocking the smtp port 25 that why i couldn't send any email(even when i opened a ticket they mentioned that the port was open).
I changed the hosting provider and now all seems good thank you will!
